fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -17,25 +17,43 @@
 class ConditionalSelectionRate(ClassificationMoment):
     """Generic fairness moment for selection rates.
 
-    This serves as the base class for both :class:`DemographicParity`
-    and :class:`EqualizedOdds`. The two are distinguished by
-    the events they define, which in turn affect the
-    `index` field created by :meth:`load_data()`.
+    This serves as the base class for :class:`DemographicParity` :class:`EqualizedOdds`, and
+    others. All subclasses can be used as difference-based constraints or ratio-based constraints.
+    
+    Difference-based constraints are written with a difference bound :math:`\\epsilon`.
+    Ratio-based constraints have a similar :math:`\\epsilon`, referred to as ratio bound slack
+    below. Additionally, they require a ratio bound. Constraints are set up for each group
```

Comment:
"ratio bound slack below" ?

Reply:
What do you mean by this?

Reply:
I was asking what was meant by "ratio bound slack below" because I didn't understand it.

Reply:
I'm unsure whether this would be fixed from your perspective by saying
`below referred to as *ratio bound slack*`
OR
is it unclear what ratio bound slack is and you'd like me to elaborate on that?

Reply:
I've now reparsed, and the 'below' is referring to the _text_ below. Now it makes sense, but you might want to rephrase slightly.